### PR TITLE
CLDR-17133 CLDRModify -fQ. update annotations from TTS name, some annotations got improperly merged

### PR DESCRIPTION
**Pull request by DraganBesevic**
CLDR-17133

- [x] This PR completes the ticket.

CLDRModify no arguments:
- No changes

CLDRModify -fQ:
- Updated TTS names to annotations to remove the superfluous entries
- Had to manually restore draft="unconfirmed"  status that got removed for some regular keyword entries in 4 files: bew.xml, cv.xml, kab.xml, to.xml

CLDRModify -fP:
- No changes

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Review by pedberg-icu — CHANGES_REQUESTED** [review]
This still has the problem of removing the draft="unconfirmed" status for the regular entry when both the regular and tts entries have draft="unconfirmed". That is non-starter. will check with Tom.

**Review by pedberg-icu — APPROVED** [review]

File: common/annotations/bew.xml [hunk 1]
```diff
@@ -15,47 +15,47 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="😀" type="tts" draft="unconfirmed">muka girang</annotation>
 		<annotation cp="😃" draft="unconfirmed">lèbar | muka | nyengir | seneng | sumringah</annotation>
 		<annotation cp="😃" type="tts" draft="unconfirmed">muka nyengir lèbar</annotation>
-		<annotation cp="😄" draft="unconfirmed">girang | ketawa | mata ketutup | muka | seneng</annotation>
+		<annotation cp="😄">girang | ketawa | mata ketutup | muka | muka girang mata ketutup | seneng</annotation>
```
Comment (btangmu) [inline common/annotations/bew.xml]: @DraganBesevic @pedberg-icu I think I see why this happens: `CLDRModify.CLDRFilter.replace` is supposed to be called with `fullpath` not just the path gotten by looping through the CLDRFile

**Comment by pedberg-icu** [comment]
> This still has the problem of removing the draft="unconfirmed" status for the regular entry when both the regular and tts entries have draft="unconfirmed". That is non-starter. will check with Tom.

Fixed that manually before merging.

File: common/annotations/bew.xml (hunk 1 above)
Comment (btangmu) [reply common/annotations/bew.xml]: https://github.com/unicode-org/cldr/pull/3307

**Pull request merged**